editor/src/clj/editor/collection_common.clj

```diff
@@ -159,15 +159,14 @@
                                         (comp (mapcat second)
                                               (util/distinct-by (comp resource/proj-path :resource)))
                                         component-property-infos)
-        game-object-instance-data {:resource build-resource
+        game-object-build-resource (:resource game-object-build-target)
+        game-object-instance-data {:resource game-object-build-resource
                                    :pose pose
                                    :property-deps go-prop-dep-build-targets
                                    :instance-msg (protobuf/assign-repeated instance-desc-with-go-props
-                                                   :component-properties component-property-descs)}
-        build-target (assoc game-object-build-target
-                       :resource build-resource
-                       :game-object-instance-data game-object-instance-data)]
-    (bt/with-content-hash build-target)))
+                                                   :component-properties component-property-descs)}]
+    (assoc game-object-build-target
+      :game-object-instance-data game-object-instance-data)))
```

Comment:
No need to recalculate the `content-hash` of the `GameObject$PrototypeDesc` - we're just tacking on instance data which will result in a `GameObject$InstanceDesc` binary.